src/config/ConfigConnect.js

```diff
@@ -0,0 +1,16 @@
+module.exports.database = "DB-Battleship";
```

Comment:
filename shoudl be "ConnectionConfig" or something like that, "ConfigConnect" is not meaningful to me

Reply:
even "DatabaseConnection" sounds better because the file is in the "config" folder